src/ssl.c

```diff
@@ -22029,10 +22032,18 @@ const char* wolfSSL_lib_version(void)
 }
 
 #ifdef OPENSSL_EXTRA
+#ifdef WOLFSSL_QT
```

Comment:
This also seems like something that should be switched based on OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT.

Reply:
Guard by OPENSSL_VERSION_NUMBER